tests/test_cli_compile.py

```diff
@@ -669,6 +676,26 @@ def test_annotate_option(pip_conf, runner, option, expected):
     assert out.exit_code == 0
 
 
+def test_setuptools_annotation(pip_conf, runner):
```

Comment:
Could you adjust `test_command_line_setuptools_read` instead, because it looks pretty similar?

Reply:
You got it.